WordPressCom-Stats-iOS.podspec

```diff
@@ -16,25 +16,14 @@ Pod::Spec.new do |s|
   s.social_media_url   = "http://twitter.com/WordPressiOS"
   s.platform     = :ios, "9.0"
   s.source       = { :git => "https://github.com/wordpress-mobile/WordPressCom-Stats-iOS.git", :tag => s.version.to_s }
-  # s.source_files  = "WordPressCom-Stats-iOS", "WordPressCom-Stats-iOS/**/*.{h,m,swift}"
-  # s.private_header_files = "WordPressCom-Stats-iOS/Private/*.h"
+  s.source_files  = "WordPressCom-Stats-iOS", "WordPressCom-Stats-iOS/**/*.{h,m,swift}"
+  s.private_header_files = "WordPressCom-Stats-iOS/Private/*.h"
+  s.resource_bundle = { 'WordPressCom-Stats-iOS' => ['WordPressCom-Stats-iOS/UI/*.storyboard', 'WordPressCom-Stats-iOS/UI/*.xib', 'WordPressCom-Stats-iOS/Resources/*.otf', 'WordPressCom-Stats-iOS/Resources/*.png'] }
   s.exclude_files = "WordPressCom-Stats-iOS/Exclude"
   s.prefix_header_file = "WordPressCom-Stats-iOS/WordPressCom-Stats-iOS-Prefix.pch"
 
   s.requires_arc = true
```

Comment:
I think we may also wish to specify Swift 2.3 in this podspec for now to be totally correct. I think this works: 

```ruby
s.pod_target_xcconfig = { 'SWIFT_VERSION' => '2.3' }
```